Marketplace/CreateUIDefinition.json

```diff
@@ -0,0 +1,258 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+    "handler": "Microsoft.Azure.CreateUIDef",
+    "version": "0.1.2-preview",
+    "parameters": {
+        "basics": [
+            {
+                "name": "appServicePlan",
+                "type": "Microsoft.Common.Section",
+                "label": "App Service Plan",
+                "elements": [
+                    {
+                        "name": "appServicePlanMessage",
+                        "type": "Microsoft.Common.TextBlock",
+                        "visible": true,
+                        "options": {
+                            "text": "The app service used for the InRule execution service needs to be added to an app service plan. You can either create a new app service plan or select and existing app service plan to use."
+                        }
+                    },
+                    {
+                        "name": "createOrUpdateAppServicePlan",
+                        "type": "Microsoft.Common.CheckBox",
```

Comment:
Every input needs a tooltip. Refer to the ones I've added here for consistency's sake: https://github.com/InRule/DynamicsCRM/pull/323/files?diff=split&w=1